runtime/doc/filetype.txt

```diff
@@ -647,21 +649,25 @@ Variables:
                           empty. Enabled by default. Set |FALSE| to enable soft
                           wrapping.
 
-To use Nvim as a manpager: >
+To use Nvim as a manpager: >sh
      export MANPAGER='nvim +Man!'
 
 Note that when running `man` from the shell and with that `MANPAGER` in your
 environment, `man` will pre-format the manpage using `groff`. Thus, Nvim
 will inevitably display the manual page as it was passed to it from stdin. One
 of the caveats of this is that the width will _always_ be hard-wrapped and not
-soft wrapped as with `g:man_hardwrap=0`. You can set in your environment: >
+soft wrapped as with `g:man_hardwrap=0`. You can set in your environment: >sh
      export MANWIDTH=999
 
 So `groff`'s pre-formatting output will be the same as with `g:man_hardwrap=0` i.e soft-wrapped.
 
 To disable bold highlighting: >
      :highlight link manBold Normal
 
+An alternative to using `MANPAGER` in shell can be redefined `man`, for example: >sh
+     man() {
+          nvim "+hide Man $1"
```

Comment:
Note that by doing this you'll most probably not have man completions from your shell (or you'll have to add it manually)

Reply:
Example for bash (5.2.21(1)-release): 

![image](https://github.com/neovim/neovim/assets/11404453/da5540db-cdd3-40ab-a4ed-53ecf7919e36)